recipes/calceph/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "3.5.1" %}
+
+package:
+  name: calceph
+  version: {{ version }}
+
+source:
+  url: https://www.imcce.fr/content/medias/recherche/equipes/asd/calceph/calceph-{{ version }}.tar.gz
+  sha256: 33cc0be1b8ffb647aff9d3ac1cf025e460451e00144050d3bbc4f13bceb11c1d
+
+build:
+  number: 0
+  run_exports:
+    # SO name changes at minor rev bumps.
+    - {{ pin_subpackage('calceph', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make                # [unix]
+
+test:
+  commands:
+    - test -f $PREFIX/include/calceph.h                           # [unix]
+    - test -f $PREFIX/lib/libcalceph${SHLIB_EXT}                  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\calceph.h exit 1   # [win]
+    - if not exist %PREFIX%\\Library\\lib\\libcalceph.lib exit 1  # [win]
```

Comment:
You're also shipping binaries! Do you have a simple example that you could test one with? Or just something like `calceph_inspector --help`

Reply:
To be honest I have no idea what those binaries do. I have been using calceph only as a library.